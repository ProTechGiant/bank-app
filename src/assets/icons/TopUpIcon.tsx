import Svg, { Path } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export function TopUpIcon({ width = 20, height = 20, color = "#2E2E2E" }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 20 20" fill={color} xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M3.33398 9.99967L4.50898 11.1747L9.16732 6.52467V16.6663H10.834V6.52467L15.484 11.183L16.6673 9.99967L10.0007 3.33301L3.33398 9.99967Z"
        fill={color}
      />
    </Svg>
  );
}
