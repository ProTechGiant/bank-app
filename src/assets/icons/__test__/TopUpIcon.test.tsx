import { create } from "react-test-renderer";

import { TopUpIcon } from "../TopUpIcon";

describe("icon/TopUpIcon", () => {
  it("renders a TopUpIcon", () => {
    const result = create(<TopUpIcon />);

    expect(result).toMatchSnapshot();
  });
});
