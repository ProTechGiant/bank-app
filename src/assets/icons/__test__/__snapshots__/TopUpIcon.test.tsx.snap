// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/TopUpIcon renders a TopUpIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="20"
  bbWidth="20"
  fill="#2E2E2E"
  focusable={false}
  height={20}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 20,
        "width": 20,
      },
    ]
  }
  vbHeight={20}
  vbWidth={20}
  width={20}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={
      {
        "payload": 4281216558,
        "type": 0,
      }
    }
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M3.33398 9.99967L4.50898 11.1747L9.16732 6.52467V16.6663H10.834V6.52467L15.484 11.183L16.6673 9.99967L10.0007 3.33301L3.33398 9.99967Z"
      fill={
        {
          "payload": 4281216558,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
